### ISSUE TITLE
find_learning_rate is not saving the data as the header indicated

### ISSUE THREAD

**Issue by symeneses** [flairNLP/flair#2612]
**Describe the bug**
Ploting the learning rate is creating an empty plot as the data is not what is expected from the `plot_learning_rate` function.
The **TUTORIAL_8_MODEL_OPTIMIZATION** is also missing in the v0.10 documentation.

**To Reproduce**
Example form [tutorial](https://github.com/flairNLP/flair/blob/v0.9/resources/docs/TUTORIAL_8_MODEL_OPTIMIZATION.md) modified.
```python
from flair.datasets import WNUT_17
from flair.embeddings import TokenEmbeddings, WordEmbeddings, StackedEmbeddings
from flair.trainers import ModelTrainer
from typing import List
from torch.optim.adam import Adam

# 1. get the corpus
corpus = WNUT_17().downsample(0.1)
print(corpus)

# 2. what tag do we want to predict?
tag_type = 'ner'

# 3. make the tag dictionary from the corpus
tag_dictionary = corpus.make_tag_dictionary(tag_type=tag_type)
print(tag_dictionary.idx2item)

# 4. initialize embeddings
embedding_types: List[TokenEmbeddings] = [
    WordEmbeddings('glove'),
]

embeddings: StackedEmbeddings = StackedEmbeddings(embeddings=embedding_types)

# 5. initialize sequence tagger
from flair.models import SequenceTagger

tagger: SequenceTagger = SequenceTagger(hidden_size=256,
                                        embeddings=embeddings,
                                        tag_dictionary=tag_dictionary,
                                        tag_type=tag_type,
                                        use_crf=True)

# 6. initialize trainer
trainer: ModelTrainer = ModelTrainer(tagger, corpus)

# 7. find learning rate
learning_rate_tsv = trainer.find_learning_rate('resources/taggers/example-ner', optimizer=Adam, iterations=10)

# 8. plot the learning rate finder curve
from flair.visual.training_curves import Plotter
plotter = Plotter()
plotter.plot_learning_rate(learning_rate_tsv)
```

**Expected behavior**
The loss vs learning plot according to the data saved in `resources/taggers/example-ner/learning_rate.tsv`.

**Screenshots**
![grafik](https://user-images.githubusercontent.com/12284951/151701064-b8ab469d-26aa-4ff2-bff6-eecc549f1e55.png)

**Environment (please complete the following information):**
 - Flair v 0.10

I could create a PR to fix this and add the documentation. I would like to confirm first if the tutorial is missing for a different reason.
